iphone/TestModule/Classes/TestModule.h

```diff
@@ -0,0 +1,17 @@
+//
+//  TestModule.h
+//  TestModule
+//
+//  Created by Hans Knöchel on 11.04.17.
+//  Copyright © 2017 Hans Knöchel. All rights reserved.
+//
+
```

Comment:
Can we use standard comments here as well and all other places where required ? Thanks!